### PR TITLE
Report list loading without details

### PR DESCRIPTION
**Pull request by bjoernricks**
**Checklist**:

- [x] Tests
- [x] [CHANGELOG](https://github.com/greenbone/gsa/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

File: gsa/src/gmp/commands/reports.js [hunk 1]
```diff
@@ -59,6 +59,16 @@ class ReportsCommand extends EntitiesCommand {
       filter,
     });
   }
+
+  get(params, options) {
+    return super.get(
+      {
+        details: 0, // ensure to request no details by default
+        ...params,
+      },
```
Comment (swaterkamp) [inline gsa/src/gmp/commands/reports.js]: Changing the order of these two lines would actually ensure details: 0. In this order, details might be overwritten if they are set in params. I'm not 100% sure, wat we want here...

Comment (bjoernricks) [reply gsa/src/gmp/commands/reports.js]: the api may decide to request reports with details like `gmp.reports.get({details: 1, filter=...});`

Comment (swaterkamp) [reply gsa/src/gmp/commands/reports.js]: In that case it's the wrong order in gmp.results.get() ?

Comment (bjoernricks) [reply gsa/src/gmp/commands/reports.js]: yes you are right. gmp.results.get doesn't allow for not getting the details. IMHO it should be changed.

Comment (swaterkamp) [reply gsa/src/gmp/commands/reports.js]: Ok, would you like to put it here as well, or do we need another PR for that?

Comment (bjoernricks) [reply gsa/src/gmp/commands/reports.js]: Another PR because it isn't related

**Pull request merged**